docs/src/reference/grid.md

```diff
@@ -2,4 +2,8 @@
 DocTestSetup = :(using Ferrite)
 ```
 
-# Grid
\ No newline at end of file
+# Grid
+```@docs
+Ferrite.full_neighborhood
+Ferrite.faceskeleton
```

Comment:
If you want to do adaptive stuff, you need to compute entities along the faces of the elements, but you need to iterate over all faces without duplication, because otherwise things will cancel out each other (e.g. due to the normal within the error formulation) 

So, what you can do with it is something like this

```julia
for face in faceskeleton(grid)
    reinit!(fv,face,grid)
    neighbors = full_neighborhood(grid,face)
    for neighbor in neighbors
        reinit!(fv_neighbors,neighbor,grid)
        for qp in 1:getnquadpoints(fv_neighbors)
            gradu_jump = (function_gradient(fv,u,qp) - function_gradient(fv_neighbor,u,qp)) # more stuff like normal and detJdA
        end
    end
end
```

what needs to be added to get a nice interface is some sort of new facevalue type that has two facevalues and reverts the list of the quadrature points for one facevalue field, since they will be flipped against each other and thus spatial dependencies could be wrong if e.g. a spatial dependent elasticity,conductivity (or whatever) matrix is needed. To visualize the problem, what the current implementation (based on local face indices) of facevalues would compare
```
                       +-----+-----+
                       | (1) x (2) |
                       |     |     |
                       |     |     |
                       | (2) x (1) |
                       +-----+-----+
                            
```

and what we actually need:
```
                       +-----+-----+
                       | (1) x (1) |
                       |     |     |
                       |     |     |
                       | (2) x (2) |
                       +-----+-----+
```
`x`'s correspond to integration points on the face